core/lib/libtomcrypt/include/tomcrypt_argchk.h

```diff
@@ -39,7 +39,7 @@ void crypt_argchk(const char *v, const char *s, int d);
 #elif ARGTYPE == 1
 
 /* fatal type of error */
-#define LTC_ARGCHK(x) assert((x))
+#define LTC_ARGCHK(x) assert((x))		// FIXME: TEE_ASSERT() or assert() ?
```

Comment:
LTC natively uses `assert()`. shall we stick to it or move it to `TEE_ASSERT()` ?